lzy/graph-executor-2/src/main/java/ai/lzy/graph/model/DirectedGraph.java

```diff
@@ -0,0 +1,22 @@
+package ai.lzy.graph.model;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class DirectedGraph {
```

Comment:
unused?

Reply:
DirectedGraph will be used as intermediate representation before creating tasks from it. Maybe I will transfer it to algo package later.